documentation/3.4.0/setup/docker/stand-alone/mac-os/README.md

```
@@ -0,0 +1,233 @@
+
+> **Warning:** Use this generator only immediately after the initial system setup and before any normal user accounts are created through the portal. It should not be used under any circumstances thereafter.
+
+```
+docker exec -i citus_master psql -U postgres -d user < insert_sample_data.sql
+```
+
+After successfully running the script mentioned above, the following user accounts will be created and available for login:
+
+| Email ID               | Password   | Role                    |
+| ---------------------- | ---------- | ----------------------- |
+| mallanagouda@gmail.com | Password1@ | State Education Officer |
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)